spec/Cabin/SingleSeat.vspec

```diff
@@ -76,50 +74,56 @@ Height:
 
 Cushion:
   type: branch
-  description: Cushion (leg support) signals.
+  description: Describes signals related to the cushion/base of the seat.
+  comment: Cushion is here considered as the part of the seat that supports the thighs.
+           Additional cushions (if any) for support of lower legs is not covered by this branch.
 
-Cushion.Height:
-  datatype: uint16
+Cushion.Tilt:
```

Comment:
I'm not sure if really the cushion/seating has a tilt or if it's rather the seat as a whole (at least in my Bimmer if I change the tilt, the backrest also moves/tilts).

Reply:
In my view it all depends on how you define the signals. We can state that tilt concerns the whole seat, but then the  question is if the backrest recline signal should change as well when you do tilt, or remain unchanged. 

Example; assume we start with this setting:

- SeatingTilt = 0 (measure against horizontal axis)
- BackrestRecline = 0 (measured against vertical axis)

If we now change SeatingTilt to 10, and both seating and backrest rotate 10 degrees, should then "BackrestRecline" still have the value 0? Or shall it state 10 degrees? I.e. what shall the reference of "BackrestRecline" be? Vertical axis? Or compared to current seat tilt with 90 degrees offset? (I.e. if angle between seating and backrest is 90 degrees, then recline is considered to be 0 degrees)

Reply:
Yes - I see that definition "issue". Your proposal would work. 
On the other hand the BackrestRecline could be also measured against the "surface axis" of the seating and the seat has an overall tilt.
Both solutions would be valid. Question is, what is the situation in all(?)/most of the cars. I cannot remember a car where you could tilt the seating without also "moving" the backrest, i.e. the backrest keeping its current angle with respect to the seating.